src/main/java/carpetaddonsnotfound/mixins/FlowerBlock_ReplaceFlowersMixin.java

```diff
@@ -15,22 +14,22 @@
 
 @Mixin(FlowerBlock.class)
 public abstract class FlowerBlock_ReplaceFlowersMixin extends Block {
-  FlowerBlock_ReplaceFlowersMixin() {
-    super(Settings.of(Material.PLANT));
+  public FlowerBlock_ReplaceFlowersMixin(Settings settings) {
+    super(settings);
```

Comment:
Yeah was wandering if it there was a nicer way of doing it, good job!

Reply:
I let the IDE do the work 😛